src/entities/room.js

```diff
@@ -0,0 +1,29 @@
+const BaseEntity = require('./base')
+
+class Room extends BaseEntity {
+    set title(title) {
+        if (!title) throw new Error('Room is required!')
+        this.attr.title = title.trim().toLowerCase()
+    }
+
+    get title() {
+        return this.attr.title.trim().toLowerCase()
```

Comment:
Why do you trim().toLowerCase() the title here if it was trimmed/LowerCased when it was saved?

Reply:
it is a crutch to make sure data is correct :)